src/emotion-styled-macro.d.ts

```diff
@@ -0,0 +1,180 @@
+declare module '@emotion/styled/macro' {
```

Comment:
is there a reason this shorter workaround didn't work? https://github.com/emotion-js/emotion/issues/933#issuecomment-447831850

Reply:
that comment was made 4 hours ago! I'll try it 😄